docs/source/technical/vigasp.md

```diff
@@ -0,0 +1,20 @@
+# Overview: 
+VIGAS/P stands for Veteriary Institute Genome Analysis System/Platform. VIGASP uses IRIDA for data management, data access control, analysis and storing the results. IRIDA uses Galaxy to execute the pipelines. 
+
+# Accessing VIGAS and logging in
+VIGAS is accessible only from official VI Laptops or through VI-VPN. 
+Those who want to get user names has to send email to Jeevan (jeevan.karloss.antony-samy@vetinst.no) with following details: 
+1. Explanation: Why do you need access to VIGAS
+2. Preferred user name and mobile number
+3. List of species data do you want to have access for. We wil talk to the managers of those projects in VIGAS and give you access to those data/metadata/results.
+
+# Data management and access control 
+In IRIDA, data (Fastq files, meta data and results) are organized as projects (species bucket). No data can stand alone. F. ex. All salmonella data are part of "Salmonella spp" project and all Listeria data are part of "Listeria spp" project. Users can create projects where they can share the data from main project to do furether project specific analysis. Only the members of the project can access, share and see data and the results.  
+
+# Pipeline overview 
+AS of January 2024, VIGAS has 12 pipelines. 8 of them are developed in house at NVI. Project managers can set up automated pipelines with customized parameters for each project.    
+
+# Data transfer and Automated Pipeline
+Sequenced data from NVI sequencing lab are semi-automatically(so far) transferred to VIGAS using SAGA/NIRD.    
+
+# Cititig VIGASP
```

Comment:
This needs a lot of fleshing out.

1. You need a screen shot on how the project list looks
2. You need a section, a screenshot and an explanation on how to run an analysis, including when you should save to project and linelist
3. You need an explanation of what the linelist is, and how to use it. This includes the fact that you can filter and sort on the linelist
4. You need a bit on how to filter columns in the linelist, and how to make templates
5. You need to tell people how to look at their analyses results, both from the sample, and from the analyses list. 
6. You need a separate page on what pipelines we have, and what they produce

And that is what I can think of right now.

Also include links to the main irida manual.